build.gradle.kts

```diff
@@ -48,7 +48,17 @@ dependencies {
         create(properties("platformType"), properties("platformVersion"))
 
         // Plugin Dependencies. Uses `platformBundledPlugins` property from the gradle.properties file for bundled IntelliJ Platform plugins.
-        bundledPlugins(properties("platformBundledPlugins").map { it.split(',') })
+        // starting from 2024.3, all json related code is know on its own plugin
+        val platformBundledPlugins =  ArrayList<String>()
+        platformBundledPlugins.addAll(properties("platformBundledPlugins").map { it.split(',').map(String::trim).filter(String::isNotEmpty) }.get())
+        /*
+         * platformVersion check for JSON breaking changes since 2024.3
+         */
+        if (prop("platformVersion").startsWith("25")) {
```

Comment:
that means that for each new major EAP version, like 2026 next year, we must change this line.
also, currenlty the build is based on 2023.3 (as per gradle.properties file), but we must add json if this will change to 2024.3 or above in the future.